gear-programs/erc20-relay/app/src/service.rs

```diff
@@ -57,23 +32,12 @@ pub struct EthToVaraEvent {
     pub receipt_rlp: Vec<u8>,
 }
 
-#[derive(Encode, TypeInfo)]
-#[codec(crate = sails_rs::scale_codec)]
-#[scale_info(crate = sails_rs::scale_info)]
-enum Event {
```

Comment:
`indexer` depends on this event, so I think it belongs to a `historical_proxy` now